frontend/pages/admin/charities/index.tsx

```diff
@@ -27,7 +65,32 @@ const AdminCharities = () => {
             Create
           </Button>
         </Stack>
+
+        <SimpleTable
+          rows={charities ?? []}
+          isLoading={!charities}
+          columns={[
+            { title: 'ID', key: 'id' },
+            { title: 'Logo', key: 'logoBase64', transformValue: createLogos },
+            { title: 'Name', key: 'name' },
+          ]}
+          actions={[deleteCharityAction, editCharityAction]}
+        />
       </Box>
+
+      {selectedCharity && (
+        <CharityDeletionDialog
+          selectedCharity={selectedCharity}
+          open={openDeleteDialog}
+          handleClose={handleCloseDeleteDialog}
```

Comment:
Maybe we only need to depend on one state?
```suggestion
          selectedCharity={selectedCharity}
          open={selectedCharity !== null}
          handleClose={() => setSelectedCharity(null)}
```